i18n/zh/docusaurus-plugin-content-docs/current/performance/manage-resources-using-resource-groups.md.md

```diff
@@ -0,0 +1,626 @@
+# 使用 Resource Groups 管理资源
```

Comment:
Hi, could you rename the file name from `x.md.md` to `x.md`? Then the zh version doc can  be generated correctly. Thanks.